src/Storefront/Resources/views/storefront/base.html.twig

```diff
@@ -15,7 +15,7 @@
 {% endblock %}
 
 {% block base_body %}
-    <body class="{% block base_body_classes %}is-ctl-{% if controllerName is not empty %}{{ controllerName|lower }}{% endif %} is-act-{% if controllerAction is not empty %}{{ controllerAction|lower }}{% endif %}{% endblock %}">
+    <body class="{% block base_body_classes %}{# @deprecated tag:v6.8.0 - Will be removed, use activeRoute instead. #}is-ctl-{% if controllerName is not empty %}{{ controllerName|lower }}{% endif %}{# @deprecated tag:v6.8.0 - Will be removed, use activeRoute instead. #} is-act-{% if controllerAction is not empty %}{{ controllerAction|lower }}{% endif %} is-active-route-{% if activeRoute is not empty %}{{ activeRoute|replace({'%this%': ".", '%that%': "-"}) }}{% endif %}{% endblock %}">
```

Comment:
imo, we can put the deprecations also on a line above. wdyt @shopware/ct-storefront ?